straxen/plugins/pulse_processing.py

```diff
@@ -11,98 +11,18 @@
 __all__ += ['NO_PULSE_COUNTS']
 
 # These are also needed in peaklets, since hitfinding is repeated
-HITFINDER_OPTIONS = tuple([
-    strax.Option(
-    'hit_min_amplitude', track=True, infer_type=False,
-    default=('hit_thresholds_tpc', 'ONLINE', True),
-    help='Minimum hit amplitude in ADC counts above baseline. '
-            'Specify as a tuple of length n_tpc_pmts, or a number,'
-            'or a string like "pmt_commissioning_initial" which means calling'
-            'hitfinder_thresholds.py'
-            'or a tuple like (correction=str, version=str, nT=boolean),'
-            'which means we are using cmt.'
-    )])
-
-HITFINDER_OPTIONS_he = tuple([
-    strax.Option(
-    'hit_min_amplitude_he',
-    default=('hit_thresholds_he', 'ONLINE', True), track=True, infer_type=False,
-    help='Minimum hit amplitude in ADC counts above baseline. '
-            'Specify as a tuple of length n_tpc_pmts, or a number,'
-            'or a string like "pmt_commissioning_initial" which means calling'
-            'hitfinder_thresholds.py'
-            'or a tuple like (correction=str, version=str, nT=boolean),'
-            'which means we are using cmt.'
-    )])
+HITFINDER_DEFAULTS = {
+    'hit_min_amplitude': 'cmt://hit_thresholds_tpc?version=ONLINE&run_id=plugin.run_id',
+}
+
+HITFINDER_DEFAULTS_he = {
+    'hit_min_amplitude_he': 'cmt://hit_thresholds_he?version=ONLINE&run_id=plugin.run_id',
+}
```

Comment:
@jmosbacher can't we just get rid of this? I.e. just change the default to the url?

Reply:
Sure, i assumed whomever put it there wanted it to be in a single place so changes are synched between the two plugins that use this default value

Reply:
Removed.